library/src/main/java/io/andref/rx/widgets/ListViewCard.java

```diff
@@ -106,7 +107,8 @@ private void initializeViews(@NonNull Context context, @Nullable AttributeSet at
 
         mButton = (TextView) cardView.findViewById(R.id.button_text);
         mButton.setText(buttonText);
-        mButtonClicks = RxView.clicks(frameLayout);
+        RxView.clicks(frameLayout)
+                .subscribe(mButtonClicks);
```

Comment:
Don't like the naked subscriptions buried in the view. Need to make sure they get unsubscribed from. Maybe `takeUntil`, `RxView.detaches()`. Or something like that.